supermarq-benchmarks/supermarq/qcvv/base_experiment.py

```diff
@@ -46,6 +75,45 @@ class Sample:
     """The corresponding data about the circuit that is needed when analyzing results
     (e.g. cycle depth)."""
 
+    def _json_dict_(self) -> dict[str, Any]:
+        """Converts the sample to a json-able dictionary that can be used to recreate the
+        sample object.
+
+        Returns:
+            Json-able dictionary of the sample data.
+        """
+        return {
+            "circuit": css.serialize_circuits(self.circuit),
```

Comment:
this one also doesn't need to be serialized manually
```suggestion
            "circuit": self.circuit,
```